graylog2-web-interface/src/views/spec/CreateNewDashboard.test.jsx

```diff
@@ -43,6 +43,10 @@ describe('Create a new dashboard', () => {
     PluginStore.register(new PluginManifest({}, viewsBindings));
   });
 
+  beforeEach(() => {
+    jest.setTimeout(30000);
+  });
```

Comment:
Is this fixing the problem locally for you, when you run `yarn test` multiple times? I am still receiving the same error.

`jest.setTimeout` will also work for the hole file without anything like `beforeEach()`. Do you use `beforeEach` to keep it in the scope of the `describe()`?

Reply:
It is fixing the problem up to a certain concurrency level. Naturally, we cannot fix it for any amount of parallelism, as the runtime will just increase.

I wrapped it in `beforeEach` to limit it to the smallest scope available.